plugins/prefixIds.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+exports.type = 'perItem';
+
+exports.active = false;
+
+exports.params = {
+    delim: '__'
+};
+
+exports.description = 'prefix IDs';
+
+
+var path      = require('path'),
```

Comment:
How are you running svgo in the browser? When you use string input method, path module is not needed.